examples/archive.html

```diff
@@ -104,17 +111,20 @@ <h3 id="offcanvasRightLabel">Images</h3>
                 let imageRow = document.createElement('div');
                 let image = new Image(150, 150);
                 let placeButton = document.createElement('a');
-
+                let imageArray=[];
```

Comment:
Let's name this "placedImagesArray" so we know it's just images which have been placed on the map

Reply:
I’ve renamed it to placedImagesArray